org.freecad.FreeCAD.yaml

```diff
@@ -0,0 +1,801 @@
+app-id: org.freecad.FreeCAD
+default-branch: beta
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: FreeCAD
+desktop-file-name-prefix: "(beta) "
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=network
+  - --filesystem=host # needed to make file saving work
+  - --filesystem=xdg-run/gvfs # make GnomeVFS accessible
+  - --filesystem=/run/spnav.sock:ro # allow access to spacenavd
+  - --talk-name=org.freedesktop.Flatpak # to launch openscad
+  - --env=TMPDIR=/var/tmp
```

Comment:
most things seem to work but this is giving me trouble, many unit tests fail when this is set but if I manually set it to /home/adrian/.cache/tmp it all works

Reply:
run unit tests with
```
org.freecad.FreeCAD -c --run-test 0
```

Reply:
solved, it was a faulty openscad test that messed up many subsequent tests because it left cwd pointing to an nonexistent directory